assets/resources/scss/_about.scss

```diff
@@ -0,0 +1,55 @@
+.about {
```

Comment:
Where do you import these scss files. You already hace css file. Pick one method.

Reply:
The _*.scss files are imported in styles.scss and compiled automatically into styles.css as a public deployment css resource.